boltdesignsystem/bolt
# DS-767 Remove usage of image component from card

## Jira

https://pegadigitalit.atlassian.net/browse/DS-767

## Summary

Support for image element was added.

## Details

- Support for the `Image Element` is added,
- Backwards compatibility with the `Image Component`
- `Card` schema was updated to point to the `Image Element`  instead of the `Image Component`,
- `Image Component` props were replaced with `Image Element` props in the pattern lab

## How to test

Pull the branch. Check if by passing `image element`  in the Card component to the **media: { image: image**
```
 {% set image %}
  {% include '@bolt-elements-image/image.twig' with {
    attributes: {
      alt: 'Image alt text',
      src: '/images/placeholders/landscape-16x9-mountains.jpg',
    },
  } only %}
{% endset %}

{% include '@bolt-components-card-replacement/card-replacement.twig' with {
  media: {
    image: image,
  },
  ...
  } only %}
  ```
  renders the image element instead of the `<bolt-image/>` component

## Release notes

When adding an image to the media inside Bolt Card component, please use a renderable image element for `media.image` instead of structured data.
